### ISSUE TITLE
Empty legend box appearing in stress plots (v2)

### ISSUE THREAD

**Issue by normanrichardson** [robbievanleeuwen/section-properties#94]
**Describe the bug**

Empty legend box appearing in stress plots.

**To Reproduce**
```python
import sectionproperties.pre.sections as sections
from sectionproperties.analysis.cross_section import Section

geometry = sections.angle_section(d=150, b=90, t=12, r_r=10, r_t=5, n_r=8)
geometry.create_mesh(mesh_sizes=[20])
section = Section(geometry)

section.calculate_geometric_properties()
section.calculate_warping_properties()
stress_post = section.calculate_stress(N=10e3)

stress_post.plot_stress_n_zz()
```
<div align ='center'>
<img src="https://user-images.githubusercontent.com/14271499/134745125-31f343c5-5123-47cf-9ae0-92d88333259f.png"  width="450" height="450" />
</div>

**Expected behavior**

Legend box removed.

**Comment by connorferster** [comment]
Thanks @normanrichardson, I will investigate this.

**Comment by normanrichardson** [comment]
No worries, it's not a big issue, but I saw it and thought I better add it before I forget. Without going into the source I resolved it with `matplotlib.pyplot.rcParams.update({"legend.frameon": False})`.

**Comment by connorferster** [comment]
Fixed in #106
